resources/usr/local/etc/sphinx/configs-available/everything.sphinx.conf

```diff
@@ -172,6 +179,8 @@ source vanilla_dev_KnowledgeArticle {
     sql_attr_uint = knowledgeCategoryID
     sql_attr_uint = insertUserID
     sql_attr_uint = updateUserID
+    sql_attr_string = locale
+    sql_attr_string = siteSectionGroup
```

Comment:
Since we add one more field for future usage `KnowledgeBaseID` lets set `sql_attr_uint = KnowledgeBaseID` as well